common/src/main/java/com/wynntils/models/quests/QuestInfo.java

```diff
@@ -163,7 +163,7 @@ public static List<Component> generateTooltipForQuest(QuestInfo questInfo) {
         tooltipLines.add(Component.literal(""));
         // We always parse level as one, so check if this mini-quest does not have a min combat level
         if (!questInfo.isMiniQuest || questInfo.additionalRequirements.isEmpty()) {
-            tooltipLines.add((Models.Character.getXpLevel() >= questInfo.getLevel()
+            tooltipLines.add((Models.Experience.getXpLevel() >= questInfo.getLevel()
                             ? Component.literal("✔").withStyle(ChatFormatting.GREEN)
```

Comment:
The QuestModel needs a new dependency as well...

Reply:
This is actually a bit of a pain. :-/ 

I need to think if I can do some redesign so we can get rid of this inter-model dependency stuff. I have some ideas that might work.

Reply:
@kristofbolyai This still need fixing.